packages/sign_in_with_apple/ios/Classes/SwiftSignInWithApplePlugin.swift

```diff
@@ -95,13 +115,11 @@ public class SwiftSignInWithApplePlugin: NSObject, FlutterPlugin {
 
             default:
                 result(FlutterMethodNotImplemented)
-
-                return
             }
         } else {
             result(
                 FlutterError(
-                    code: "ERR",
+                    code: "not-supported",
```

Comment:
Should we group all error codes in a single location as enum constants? Saw that in another package and makes it great to check what incoming data to expect in the Dart side

Reply:
Yeah sounds good 👍